### ISSUE TITLE
broken submodules (breaks glide and dep)

### ISSUE THREAD

**Issue by tobowers** [attic-labs/noms#3782]
When trying to use glide or the "official" go dep manager I get error message similar to `76400c6b710112a3ec35cb963c3eea1bb13d9dd3: unexpected error while defensively updating submodules: : command failed: [git submodule update --init --recursive]: exit status 1` (glide is slightly different, but also fails on the submodule update)

Performing the same command in the directory:

```
$ git submodule update --init --recursive
No submodule mapping found in .gitmodules for path 'vendor/gx/ipfs/QmcRKRQjNc2JZPHApR32fbkZVd6WXG2Ch9Kcy7sPxuAJgd/cbor/test-vectors'
```

**Comment by fische** [comment]
`go get` also fails because of submodules:
```
$ go get -u github.com/attic-labs/noms
# cd $GOPATH/src/github.com/attic-labs/noms; git submodule update --init --recursive
fatal: No url found for submodule path 'vendor/gx/ipfs/QmcRKRQjNc2JZPHApR32fbkZVd6WXG2Ch9Kcy7sPxuAJgd/cbor/test-vectors' in .gitmodules
package github.com/attic-labs/noms: exit status 128
```

**Comment by tobowers** [comment]
@fische I just opened a PR to fix it #3784

**Comment by senorsmile** [comment]
Just trying the project out for the first time.  This is hitting me too trying ```go get``` .

What's the work around until someone merges?
